frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/DiscoverAction.java

```diff
@@ -51,12 +55,14 @@ public LambdaLogger log() {
   }
 
   @Override
-  // TODO (#1472): extract a backgroundable for the heavy lifting
-  @IgnoreUIThreadUnsafeCalls({
-      "com.virtuslab.gitmachete.backend.api.IGitMacheteRepository.discoverLayoutAndCreateSnapshot()",
-      "com.virtuslab.gitmachete.backend.api.IGitMacheteRepositoryCache.getInstance(java.nio.file.Path, java.nio.file.Path, java.nio.file.Path)"
-  })
   @UIEffect
+  public void onUpdate(AnActionEvent actionEvent) {
+    super.onUpdate(actionEvent);
+    actionEvent.getPresentation().setDescription(getNonHtmlString("action.GitMachete.DiscoverAction.description"));
```

Comment:
Unrel, to prevent:

<img width="606" alt="image" src="https://user-images.githubusercontent.com/3383210/220142725-bb63520f-67f4-4eda-a805-e2b18c331348.png">